custom_components/opnsense/switch.py

```
@@ -10,241 +11,292 @@
+        else:
+            _LOGGER.error(
+                f"Error in switch async_setup_entry. {result.__class__.__qualname__}: {result}"
+            )
+    _LOGGER.debug(f"[switch async_setup_entry] entities: {len(entities)}")
+    async_add_entities(entities)
 
 
 class OPNsenseSwitch(OPNsenseEntity, SwitchEntity):
+
     def __init__(
         self,
```

Ruff

Missing type annotation for `self` in method

(ANN101)